scripts/startup/README.md

```diff
@@ -15,8 +19,55 @@ sudo chmod +x /etc/profile.d/set_panoptes_env_vars.sh
 This will then be executed when any user logs in. The exact means
 of doing this varies on different versions of Unix/Linux.
 
-Next, we need to arrange for `su_panoptes.sh` to be executed when
-the operating system launches. First copy it to /etc:
+If you've changed the location of the PANOPTES directories (e.g. if
+log files are not stored in $PANDIR/logs), you'll need to edit that
+script accordingly.
+
+# Start PANOPTES Software on System Start
+
+We intend for a PANOPTES telescope to run automatically, including after
+reboots. We document these ways to achieve this:
+
+* `@reboot` rule in crontab of user $PANUSER
+* Execution by root during boot
+
+## Crontab
+
+[Crontab](https://linux.die.net/man/5/crontab) provides a means to run
+a command line at various times. Some versions of crontab have support
+for [`@reboot`](https://www.google.com/search?q=crontab+%40reboot) as
+a *time* at which to run a command. If your system has this feature, it
+is the easiest way to launch the PANOPTES software.
+Add these lines to your ($PANUSER) crontab by executing the command
+`crontab -e`, which will start an editor on your crontab file.
+
+```bash
+POCS=/var/panoptes/POCS
+PANLOG=/var/panoptes/logs
+
+# m h  dom mon dow   command
+@reboot              /bin/bash --login $POCS/scripts/startup/tmux_launch.sh
+*/5 *   *   *   *    /bin/bash --login python $POCS/scripts/plot_weather.py >> $PANLOG/plot_weather.log 2>&1
+11  12  *   *   *    /bin/bash --login python $POCS/pocs/utils/data.py >> $PANLOG/update_data.log 2>&1
+```
+
+If your values for POCS and PANLOG don't match those show, please
+edit them appropriately. Use the fully evaluated values (e.g. the
+value produced by `echo $POCS` and by `echo $PANLOG`), as `cron`
+does not expand variables in those lines.
+
+Save the edited file and exit the editor.
+
+Notice that each of these commands starts with `/bin/bash --login`. We
+do this because `cron` does not run the commands in a shell with all
+of the normal environment variables set, so we force that to happen.
+
+## Execution by `root`
+
+If necessary, you can modify the Linux boot scripts to start the
```

Comment:
Explain when "if necessary" applies, i.e. "if your crontab does not support the reboot directive". Of course then we will just get questions about which ones do and do not. I'm guessing it's not as simple as a version number but rather which implementation?

Reply:
We should figure out if works on ubuntu 16.04 (and maybe upcoming 18.04) and highlight that.